### PR TITLE
Remove references to 'Evaluation' #3589

### LINKED ISSUE #3589: Remove references to 'Evaluation'
Are these redundant or are they using the term 'evaluation' to refer to feedback sessions? By the way, why do we have 'test' in the code?
image

### PR DESCRIPTION
**Pull request by kanghj**
Renamed the issue to include other references to evaluations other than just on instructorHome. 

Fixes #3589

### CODE REVIEW

File: src/main/webapp/showMessage.jsp [hunk 1]
```diff
@@ -19,7 +19,7 @@
                     }
                 }
             }
-            var nextUrl = "<%=Const.ActionURIs.INSTRUCTOR_EVALS_PAGE%>";
+            var nextUrl = "<%=Const.ActionURIs.INSTRUCTOR_FEEDBACKS_PAGE%>";
             if(user!==null){
```
Comment (kanghj) [inline src/main/webapp/showMessage.jsp]: While I have modified this file and it gets tested in AllAccessControlUiTest, it does not seem to be used anywhere. Will make a new issue for this

File: src/main/java/teammates/logic/core/StudentsLogic.java [hunk 2]
```diff
@@ -79,8 +79,7 @@ public void createStudentCascadeWithoutDocument(
     
     public void createStudentCascade(StudentAttributes studentData, boolean hasDocument) 
             throws InvalidParametersException, EntityAlreadyExistsException, EntityDoesNotExistException {
-        
-        createStudentCascadeWithSubmissionAdjustmentScheduled(studentData, hasDocument);
+        studentsDb.createStudent(studentData, hasDocument);
```
Comment (damithc) [inline src/main/java/teammates/logic/core/StudentsLogic.java]: No need to cascade?

Comment (kanghj) [inline src/main/java/teammates/logic/core/StudentsLogic.java]: The cascading was done for evaluation submissions, to add a submission to/from the other peers in the same team. For feedback sessions, we currently simply treat this case as a case where there is no response from/to the new student in the team. This does not change the calculations for the values (Claimed contribution and perceived contribution) in contribution questions, as the calculations will ignore the students without a response.

Comment (kanghj) [inline src/main/java/teammates/logic/core/StudentsLogic.java]: Ah, I should change the function name.

**Pull request merged**